catalog/core/catalog-core-directorymonitor/pom.xml

```diff
@@ -379,13 +377,18 @@
                 <artifactId>maven-bundle-plugin</artifactId>
                 <configuration>
                     <instructions>
-                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <!-- Override this module's blueprint timeout to be 30 minutes instead of 5 (default) -->
+                        <Bundle-SymbolicName>${project.artifactId};blueprint.timeout:=1800000</Bundle-SymbolicName>
```

Comment:
❓ Do we need to bump this up to 30 minutes even after embedding the http libraries?

Reply:
We do if we want the `org.codice.ddf.platform.bootflag.transformerWaitTimeoutSeconds` property to actually make a difference above 5 minutes. Otherwise, the blueprint container will timeout before the bootflag timeout is hit.